sdk/storage/Azure.Storage.Common/src/Sas/SasQueryParameters.cs

```diff
@@ -40,129 +40,173 @@ public partial class SasQueryParameters
         /// access signature. 
         /// </summary>
-        public string Services => services ?? string.Empty;
+        public string Services => _services ?? string.Empty;
 
         /// <summary>
         /// Gets which resources are accessible via the shared access signature.
         /// </summary>
-        public string ResourceTypes => resourceTypes ?? string.Empty;
+        public string ResourceTypes => _resourceTypes ?? string.Empty;
 
         /// <summary>
         /// Optional. Specifies the protocol permitted for a request made with
         /// the shared access signature.
         /// </summary>
-        public SasProtocol Protocol => protocol;
+        public SasProtocol Protocol => _protocol;
 
         /// <summary>
         /// Gets the optional time at which the shared access signature becomes
         /// valid.  If omitted, start time for this call is assumed to be the
         /// time when the storage service receives the request.
         /// <see cref="DateTimeOffset.MinValue"/> means not set.
         /// </summary>
-        public DateTimeOffset StartTime => startTime;
+        public DateTimeOffset StartTime => _startTime;
 
         /// <summary>
         /// Gets the time at which the shared access signature becomes invalid.
         /// <see cref="DateTimeOffset.MinValue"/> means not set.
         /// </summary>
-        public DateTimeOffset ExpiryTime => expiryTime;
+        public DateTimeOffset ExpiryTime => _expiryTime;
 
         /// <summary>
         /// Gets the optional IP address or a range of IP addresses from which
         /// to accept requests.  When specifying a range, note that the range
         /// is inclusive.
         /// </summary>
-        public IPRange IPRange => ipRange;
+        public IPRange IPRange => _ipRange;
 
         /// <summary>
         /// Gets the optional unique value up to 64 characters in length that
         /// correlates to an access policy specified for the container, queue,
         /// or share.
         /// </summary>
-        public string Identifier => identifier ?? string.Empty;
+        public string Identifier => _identifier ?? string.Empty;
 
         /// <summary>
         /// Gets the resources are accessible via the shared access signature.
         /// </summary>
-        public string Resource => resource ?? string.Empty;
+        public string Resource => _resource ?? string.Empty;
 
         /// <summary>
         /// Gets the permissions associated with the shared access signature.
         /// The user is restricted to operations allowed by the permissions.
         /// This field must be omitted if it has been specified in an
         /// associated stored access policy.
         /// </summary>
-        public string Permissions => permissions ?? string.Empty;
+        public string Permissions => _permissions ?? string.Empty;
+
+        /// <summary>
+        /// Gets the cache control to be used in the Cache-Control response header.
```

Comment:
These doc comments aren't particularly helpful.  Can we say something about what `Cache-Control` does to let folks know why they might want to use it?

Reply:
I figured that would be overkill since these are standard HTTP response headers. 
I also followed similar convention to the doc for public properties used in BlobSasBuilder.
Do you think we should update to include link to Mozilla documentation or just copy it in?

Reply:
I think people find how some of these standard HTTP features integrate with blobs to be a little confusing.  Don't link or copy MDN.  I'd like to see just a sentence of context because not everybody knows what content disposition is and how it relates to Storage.